eval/truthfulqa/run_eval.py

```diff
@@ -333,6 +333,9 @@ def main(args):
             if "mc" in args.metrics:
                 raise ValueError("OpenAI Chat engines does not support MC metrics.")
 
+    del model
+    torch.cuda.empty_cache()
```

Comment:
oom? 👀

Reply:
yeah, I had some ooms while evaluating deepseek's 16b MoE and I realized we can jus delete the model after this point for free wins